middleware/rate-limiter.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const {
+    RateLimiterMySQL,
+    RateLimiterMemory
+} = require('rate-limiter-flexible');
+const { startsWith } = require('lodash');
+
+const { sequelize } = require('../db/models/index');
+const appData = require('../common/appData');
+const logger = require('../common/logger').child({
+    service: 'rate-limiter'
+});
+
+const sqlDialect = startsWith(process.env.DB_CONNECTION_URI, 'sqlite://')
```

Comment:
Rather than doing this check again could you try to read the value out of `databaseConfig` (see db/index.js), perhaps by exporting it, or you might be able to ask the `sequelize` instance for this.

Reply:
`sequelize.getDialect()` FTW